src/main/java/eu/darkbot/api/managers/RepairAPI.java

```diff
@@ -0,0 +1,27 @@
+package eu.darkbot.api.managers;
+
+import eu.darkbot.api.API;
+
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Optional;
+
+public interface RepairAPI extends API {
+
+    int getDeathsAmount();
+
+    boolean isDestroyed();
+
+    /**
+     * Tries to repair ship with given repair option
+     *
+     * @throws IllegalStateException if ship is already repaired
+     */
+    void tryRevive(int repairOption) throws IllegalStateException;
+
+    Collection<Integer> getAvailableRepairOptions();
+
+    Optional<String> getLastDestroyerName();
+
+    Optional<LocalDateTime> getLastDeathTime();
```

Comment:
I feel like this may be a bit of an abuse of Optional, this is simple enough that we can just return nulls

Also, use Instant for the time datatype